src/beast/beast/nudb/win32_file.h

```diff
@@ -42,6 +42,9 @@
 # ifndef STRICT
 #  define STRICT
 # endif
+# ifndef WIN32_LEAN_AND_MEAN
```

Comment:
@rec Yeah, its an unfortunate consequence of Microsoft's sloppy headers. If we define it for the whole project, then code which needs more than what's in the lean and mean windows headers will break.